### PR TITLE
docs: automation of upgrades

### PR DESCRIPTION
**Pull request by nqb**
# Description
Documentation part of #6583 

# Impacts
No

# Code / PR Dependencies
PR #6583 

# Delete branch after merge
(REQUIRED)
YES

### CODE REVIEW

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 1]
```diff
@@ -54,6 +56,7 @@ You can take a backup using:
 
 === Disable monit alerts (only if you have monit installed)
 
+NOTE: This step is not necessary if you are doing an <<PacketFence_Installation_Guide.asciidoc#_automation_of_upgrades,automated upgrade>>.
```
Comment (julsemaan) [inline docs/PacketFence_Upgrade_Guide.asciidoc]: I'd say that its available when upgrading from 11.0 or above here to help make it clearer

File: docs/installation/automation_of_upgrades.asciidoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+////
+
+//== Automation of upgrades
+
+This section covers automation of upgrades available since PacketFence 11.0.0.
+
+=== Assumptions and limitations
+
+* You can perform automated upgrades on **standalone** servers only. Cluster upgrades are not supported.
```
Comment (julsemaan) [inline docs/installation/automation_of_upgrades.asciidoc]:
```suggestion
* You can perform automated upgrades on **standalone** servers only. Cluster upgrades must use the procedure described in the Clustering Guide.
```

**Comment by extrafu** [comment]
All good for me regarding the doc.

File: docs/installation/automation_of_upgrades.asciidoc (hunk 2 above)
Comment (julsemaan) [reply docs/installation/automation_of_upgrades.asciidoc]: This says resolved but there is no change

**Comment by julsemaan** [comment]
I tested the documentation steps (there were a lot!) and it all works fine

Just 2 💅 details on my end and its good to merge

**Comment by nqb** [comment]
Should be good @julsemaan. I add one thing I missed related to minor upgrades.

**Pull request merged**